ONSdigital/eq-questionnaire-validator
# Use ajv validator

### PR Context

- Replace python Draft7schema validator to node based AJV schema validator
- Create node based Dockerfile For AJV validator
- update workflow to push newly created AJV docker image

[Companion Runner PR ](https://github.com/ONSdigital/eq-questionnaire-runner/pull/724)

Trello card : https://trello.com/c/TUoAGONd/4982-use-ajv-to-validate-json-schema-5-sp
### How to test
Ensure all tests are passing 
Make sure `make run` runs both python and AJV service
Update any valid schema with wrong values and post to `http://localhost:5001/validate` ( post `make run`), ensure the error response is sent with details of error

### Checklist
